index.html

```diff
@@ -1787,6 +1876,172 @@ <h2>
         </ol>
       </section>
     </section>
+    <section>
+      <h2>
+        File Handler Items
+      </h2>
+      <p>
+        Each <dfn data-local-lt="file handler's">file handler</dfn> represents
+        a url in the scope of the application that can handle launches with
+        file types it accepts. It has the following members:
+      </p>
+      <ul>
+        <li>[=file handler/action=]
+        </li>
+        <li>[=file handler/name=]
+        </li>
+        <li>[=file handler/accepts=]
+        </li>
+        <li>[=file handler/icons=]
+        </li>
+      </ul>
+      <p>
+        A user agent can use these members to associate the web application
+        with specific file extensions or [=mime types=] on the operating
```

Comment:
This might be a good candidate to hoist into a definition of "file type" (covering extensions + mime types) and then linking from here and elsewhere.

Reply:
I put it in a 'concepts' section. WDYT?